Updated the Chart height

Joeboyc2 opened the pull request:
On the new inky displays, the values used for the charts y height causes it to overlay the current value text when using agile_price mode
Adjusting this down makes the display more legible 

Before | After
--- | ---
![20210929_092210](https://user-images.githubusercontent.com/8202030/135352505-c6226e5d-859b-40db-897b-6d39b76bb768.jpg) | ![20210929_093102](https://user-images.githubusercontent.com/8202030/135352521-ee5bca4c-9272-401a-a49b-55288829e331.jpg)

jerbzz commented:
I'm going to (temporarily) fix this differently by changing the overall y-height for both displays. This is happening because of the current bonkers Agile prices - previously, it was rare to hit the 35p cap at all so there would only be a little bit of occlusion.
It would be nice to draw a black outline round the text too... hmmm.

jerbzz commented:
https://github.com/jerbzz/pi-eco-indicator/commit/67dd36ce6b04d3c29f28dd18f2c30f388e7ecc1a

Here ya go, if you feel like updating your repo (not to 2.1.0, it's in main but not tagged a release yet) and giving it a test, that'd be super :)

Joeboyc2 commented:
Running the new code and all looks good, no issues with running the commands, thank you for all the updates

Joe

jerbzz commented:
Hey,

So the latest code with auto scaling etc isn't tagged as a release at all, yet, because it needs testing. 

Sounds like for your error you had some of the changes but not others. 

The easiest way to make sure that you have the latest code (development version) is to delete your pi-eco-indicator directory, then

cd ~

git clone https://github.com/jerbzz/pi-eco-indicator

Then follow normal setup steps of copying config.yaml.default to config.yaml 

etc :)

Joeboyc2 commented:
ah ok,
Let me give the change above a go no I have update the base code, I'll come back to you in a min

Joeboyc2 commented:
Ok, so I have updated the code to what was in the change above, and it doesn't look as good
![20211011_081748](https://user-images.githubusercontent.com/8202030/136750704-1f82a5c7-131c-434a-8d40-b91987810bce.jpg)

so I have updated the value in line 157 from:
> graph_y_unit = (inky_display.HEIGHT / 2.5) / max_slot_value

To
> graph_y_unit = (inky_display.HEIGHT / **2.8**) / max_slot_value

and that looks a little better:
![20211011_083317](https://user-images.githubusercontent.com/8202030/136750963-9161cba1-5379-4b98-9b46-97f5a5befcc7.jpg)
